### ISSUE TITLE
Add how-to for following rss feed for tachi sources that don't support custom repos yet.

### ISSUE THREAD

**Issue by Nitrousoxide** [keiyoushi/tachiyomi-extensions-source#7]
### Source name

N/A

### Source language

N/A

### Describe your suggested feature

Add this url for an RSS feed to stay up to date on commits to the sources to the readme while folks are waiting on tachi forks to update.

https://github.com/keiyoushi/tachiyomi-extensions/commits/repo/apk.atom

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I have updated the app to version **[0.15.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.
